examples/connext_dds/network_capture/03_security/c/CMakeLists.txt

```
@@ -11,12 +11,24 @@ list(APPEND CMAKE_MODULE_PATH
 include(ConnextDdsConfigureCmakeUtils)
 connextdds_configure_cmake_utils()
 
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.1"
+        REQUIRED
+        COMPONENTS
+            core
+            security_plugins
+    )
+endif()
```

```suggestion
find_package(RTIConnextDDS
    "6.1.1"
    REQUIRED
    COMPONENTS
        core
        security_plugins
)
```
Since we call find_package(RTIConnextDDS previously when building all the examples, CMake will not enter here, therefor not loading the security_plugins. We need to remove the `if` condition.